### PR TITLE
Added Columnar Cipher in Python

### LINKED ISSUE #3805: Columnar Cipher in Python
## 💥 Proposal

Implementing columnar cipher in python. It is a form of transposition cipher. Plaintext is written in rows and ciphertext is read out by columns using the order of key.

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?

Yes

### PR DESCRIPTION
**Pull request by nidh-ish**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Implemented columnar cipher in python. It is a form of transposition cipher. The plaintext is written in rows and ciphertext is read out by columns using the order of key.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Fixes #3805

### CODE REVIEW

File: Python/cryptography/Columnar_Cipher.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
+'''
+In Columnar Cipher, we are given a plaintext and a key. The plaintext is written out in rows
```
Comment (anushkrishnav) [inline Python/cryptography/Columnar_Cipher.py]: Line too long pass your code on linter [here](http://pep8online.com) and fix the lining /styling issues

File: Python/cryptography/Columnar_Cipher.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+key given. Key is converted to its corresponding numerical(Ex A-0, B-1, C-2, ...... , Z-25).
+Cipher text is formed by reading the columns corresponding to the lowest numerical value in 
+the key to highest numerical value.
+'''
+
+''' This function takes key as argument. It then sorts
+    the key according to alphabetical order and return 
+    the relative position of each keyword.'''
+def key_to_pad(keyword):
+
```
Comment (anushkrishnav) [inline Python/cryptography/Columnar_Cipher.py:15]: Add the docstring below the function definition

File: Python/cryptography/Columnar_Cipher.py [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    for i in keyword:
+        pad.append(sorted_keyword.index(i))
+    return pad
+
+
+''' This funcion takes plaintext and keyword as input.
+    It forms a matrix where plaintext characters are
+    stored in row wise fashion.'''
+def matrix_plaintext(plaintext, keyword):
+
```
Comment (anushkrishnav) [inline Python/cryptography/Columnar_Cipher.py:28]: Add the docstring below the function definition

File: Python/cryptography/Columnar_Cipher.py [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+    # This function returns the matrix and the final plaintext
+    return plaintext, matrix
+
+
+''' This is the actual encrption function. It takes plaintext
+    and keyword as input. It then forms ciphertext by reading
+    the matrix in a column wise fashion. It used the relative
+    order of key to decide the order of chosing columns.'''
+def encryption(plaintext, keyword):
+
```
Comment (anushkrishnav) [inline Python/cryptography/Columnar_Cipher.py:68]: Add the docstring below the function definition

**Review by anushkrishnav — CHANGES_REQUESTED** [review]

**Comment by nidh-ish** [comment]
@anushkrishnav I have made the changes. Please have a look.

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**